aspy/refactor_imports/classify.py

```diff
@@ -31,6 +31,15 @@ def _due_to_pythonpath(module_path: str) -> bool:
     return mod_dir in _pythonpath_dirs()
 
 
+def _has_path_prefix(path: str, prefix: str) -> bool:
```

Comment:
perhaps make `prefix` keyword-only?

I'd also suggest a test and not no-covering

Reply:
Done: https://github.com/asottile/aspy.refactor_imports/pull/53/commits/452f0a2a791434588b783910f6c7f2d72fc402e1

The no cover pragma is no longer needed.